### ISSUE TITLE
Implement Precision-Recall Curves for What-If Tool

### ISSUE THREAD

**Issue by henningsway** [tensorflow/tensorboard#2253]
The What-If Tool seems really useful.

However for imbalanced data, additional metrics would be great. Being able to use to use Precision-Recall Curves instead of ROC would be amazing. F0.5 Score and logloss as metric would also be very much appreciated.

[The Precision-Recall Plot Is More Informative than the ROC Plot When Evaluating Binary Classifiers on Imbalanced Datasets](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0118432)

**Comment by jameswex** [comment]
Yup, we have this on our tracking list as well. It definitely is happening.
